doc/reference/releases-snap.md

```diff
@@ -0,0 +1,127 @@
+- **Extended support**: Only critical security updates are provided.
+
+(ref-releases-lts-support-current)=
+#### Currently supported
+
+The currently supported LTS releases are 5.21._z_ and 5.0._z_.
+
+- 5.21._z_ is supported until June 2029.
+  - Currently in full support phase.
+  - Updates are provided approximately every six months. This schedule is an estimate that can change based on priorities and discovered bugs.
+- 5.0._z_ is supported until June 2027.
+  - Currently in extended support phase.
+  - Updates are provided when critical security updates are needed.
+
+(ref-releases-feature)=
+### Feature releases
+
+Feature releases are pushed out more often and contain the newest features and bugfixes. Since they are less tested than LTS releases, they are **not recommended for production use**.
+
+These releases follow the format _x.y_, and they never include a point number _z_. Currently, feature releases for LXD are numbered {{current_feature_track}}._y_, with _y_ incrementing for each new release. Every two years, the latest feature release becomes an LTS release.
+
+#### Support
+
+Feature releases receive continuous updates via each new release. The newest release at any given time is also eligible for additional support through an [Ubuntu Pro](https://ubuntu.com/pro) subscription.
+
+(ref-snap)=
+## The LXD snap
+
+The recommended way to install LXD is [its snap package](https://snapcraft.io/lxd), if snaps are available for your system. A key benefit of snap packaging is that it includes all required dependencies. This allows LXD to run in a consistent environment on many different Linux distributions. Using the snap also streamlines updates through its [channels](https://snapcraft.io/docs/channels).
+
+(ref-snap-channels)=
+### About channels
+
+Each installed LXD snap follows a [channel](https://snapcraft.io/docs/channels). Channels have a track and a risk level (for example, `6/stable`). Each channel points to one release at a time, and when a new release is published to a channel, it replaces the previous one. {ref}`Updating the snap <howto-snap-updates>` then updates to that release.
+
+Use only the LTS and feature release channels for LXD. To view all available channels, run:
+
+```bash
+snap info lxd
+```
+
+(ref-snap-tracks)=
+#### Tracks
+
+LXD releases are grouped under [snap tracks](https://snapcraft.io/docs/channels#heading--tracks), such as `6` or `5.21`. For each track, there are three risk levels.
+
+(ref-snap-risk)=
+#### Risk levels
+
+Available [risk levels](https://snapcraft.io/docs/channels#heading--risk-levels) for LXD are `stable`, `candidate`, and `edge`. Use the `stable` risk level to install fully tested releases; this is the only risk level supported under [Ubuntu Pro](https://ubuntu.com/pro). The `candidate` and `edge` levels offer newer but less-tested updates, posing higher risk.
+
+(ref-snap-channels-default)=
+### The default channel
```

Comment:
This isn't quite right. There is a default "track" which is currentl "5.21", then snap command itself defaults to the "stable" channel inside that track.